book/02-git-basics/sections/undoing.asc

```diff
@@ -125,17 +126,17 @@ Changes to be committed:
 
 ----
 
-You can see that the changes have been reverted.
+Değişikliklerin geri alındığını görebilirsiniz.
 
 [IMPORTANT]
 =====
-It's important to understand that `git checkout -- <file>` is a dangerous command.
-Any local changes you made to that file are gone -- Git just replaced that file with the most recently-committed version.
-Don't ever use this command unless you absolutely know that you don't want those unsaved local changes.
+`git checkout -- <file>` komutunun tehlikeli bir komut olduğunu anlamak önemlidir.
+Bu dosyada yaptığınız tüm yerel değişiklikler kayboldu! Git, bu dosyayı en son kaydedilen sürümle değiştirdi.
+Kaydedilmemiş yerel değişiklikleri istemediğinizden kesinlikle emin olmadığınız sürece bu komutu asla kullanmayın!!!
```

Comment:
üç ünlem bütünlükteki dilin akışını bozmuş gibi duruyor, vurgu için bence seçilen kelimeler yeterli. orijinalindeki gibi nokta ile bitse daha ciddi ve profesyonel dururdu diye düşünüyorum. kişisel düşüncem olduğu için sadece bir yorum olarak yazıyorum, approve verdim zaten.

Reply:
Haklısınız. Düzeltildi.